Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -681,6 +683,31 @@ static void WriteZ24Encoder(char*& p, APIType ApiType, const EFBCopyFormat& form
   WriteEncoderEnd(p);
 }
 
+static void WriteXFBEncoder(char*& p, APIType ApiType, const EFBCopyFormat& format)
+{
+  WriteSwizzler(p, GX_CTF_XFB, ApiType);
```

Comment:
So what you need to do to enable y-scaling is modify this generated swizzler to point at the pre-scaled texture coordinates. This might be (will often be) somewhere between two lines.

Bilinear filtering is already enabled for higher IRs and halfscale textures, so you will need to make sure it's also enabled for these scaled xfb copies too.